### PR TITLE
Get reference id from BCF file

### PR DESCRIPTION
**Pull request by anamanolache**
Get reference id from BCF file

### CODE REVIEW

File: internal/bcf/bcf_test.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	"strings"
+	"testing"
+)
+
+func TestGetReferenceId(t *testing.T) {
+	testCases := []struct {
+		file   string
+		name   string
+		id     int
+		errMsg string
```
Comment (kemp-google) [inline internal/bcf/bcf_test.go]: Testing error messages is generally a bad idea.
If you care about the error, your clients probably will too.  In that case, we should have an error type and a function like IsNotFoundError(err) that does the appropriate type assertion (or you can export a simple value if the error is not dynamic - as in this case)

Comment (anamanolache) [reply internal/bcf/bcf_test.go]: I don't care about the error message, just checking that it returned an error should be enough.

File: internal/bcf/bcf_test.go [hunk 2]
```diff
@@ -46,7 +45,7 @@ func TestGetReferenceId(t *testing.T) {
 			}
 			defer r.Close()
 
-			if id, err := GetReferenceID(r, tc.name); err != nil && !strings.Contains(err.Error(), tc.errMsg) {
+			if id, err := GetReferenceID(r, tc.name); err != nil && (err.Error() != "") == tc.err {
```
Comment (kemp-google) [inline internal/bcf/bcf_test.go]: (err == nil) == tc.err

Comment (anamanolache) [reply internal/bcf/bcf_test.go]: sorry, I didn't test it. done

**Pull request merged**